apps/fortuna/src/keeper.rs

```diff
@@ -427,25 +433,39 @@ pub async fn watch_blocks(
 }
 
 /// It waits on rx channel to receive block ranges and then calls process_block_range to process them.
+#[tracing::instrument(skip_all)]
 pub async fn process_new_blocks(
     chain_state: BlockchainState,
     mut rx: mpsc::Receiver<BlockRange>,
     contract: Arc<SignablePythContract>,
     gas_limit: U256,
 ) {
+    tracing::info!("Waiting for new block ranges to process");
     loop {
-        tracing::info!(
-            "Chain: {} - waiting for new block ranges to process",
-            &chain_state.id
-        );
         if let Some(block_range) = rx.recv().await {
             process_block_range(
                 block_range,
                 Arc::clone(&contract),
                 gas_limit,
                 chain_state.clone(),
             )
+            .in_current_span()
             .await;
         }
     }
 }
+
+/// Processes the last `BACKLOG_RANGE` blocks for a chain.
```

Comment:
this comment is not accurate. We do not use the constant `BACKLOG_RANGE` but the parameter `backlog_range`